src/index.html

```diff
@@ -11,8 +12,74 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
-  <body>
-    <h1>Product cards</h1>
+
+  <!-- #endregion -->
+
+  <body
+    class="card"
+    data-qa="card"
+  >
+    <h1 class="imac">
+      <img
+        src="images/Rectangle 4.png"
```

Comment:
The image source should not be a placeholder like 'Rectangle 4.png'. Make sure to use a descriptive filename that reflects the content of the image.